features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+@owner:lilla
+@author:lilla
+
+Feature: Navigating Sidenav
+    This allows users to easily navigate and select between articles and categories within our help center site
+
+    @ui
+    @motivating
+    Scenario Outline: Default menu item state in the side navigation bar
+        Given a user looks at the <menuItem> menu item in the side navigation bar
+        Then they see that the background color is <backgroundColor>
+        And text color is <textColor>
+        And text type is  <textType>
+        And the icon color is <iconColor>
+        And the dropdown line is <dropdownLine>
+        And the arrow points <arrowDirection>
+
+        Examples:
+
+            | <menuItem>                                  | <backgroundColor> | <textColor>         | <textType> | <iconColor>         | <dropdownLine>      | <arrowDirection> |
+            | Core Category Section                       | FFFFFF            | 003B45 (dark green) | normal     | 003B45 (dark green) | FFFFFF              | N/A              |
```

Comment:
Seeing in these examples that the textcolor, text type and icon color never change - if this is the case, then they should be removed as variables and hard coded into the step where they are used

Reply:
Its best to keep the example tables as simple as possible. Also the standard for the headers is `menuItem` not `<menuItem>` not sure if cucumber would fully parse the latter

Reply:
Please apply these comments to the specs below as well